auth/src/main/java/com/firebase/ui/auth/IdpResponse.java

```diff
@@ -75,6 +96,11 @@ public Intent toIntent() {
     }
 
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    public boolean isSuccessful() {
+        return mException == null;
```

Comment:
Should this check if exception is null **and** user is non-null?

Reply:
I don't that's necessary since those two are mutually exclusive. The builder throws an NPE if the user is null and `fromError` always puts in a nonnull exception and a null user.